src/main/kotlin/org/opensearch/commons/alerting/action/GetAlertsRequest.kt

```diff
@@ -6,26 +6,30 @@ import org.opensearch.common.io.stream.StreamInput
 import org.opensearch.common.io.stream.StreamOutput
 import org.opensearch.commons.alerting.model.Table
 import java.io.IOException
+import java.util.Collections
 
 class GetAlertsRequest : ActionRequest {
     val table: Table
     val severityLevel: String
     val alertState: String
     val monitorId: String?
     val alertIndex: String?
+    val monitorIds: List<String>?
```

Comment:
If we've defined `monitorIds` as nullable here, we should also make it nullable in the constructor (or vice versa so it's consistent)

Reply:
ack. done